cmd/lncli/profile.go

```diff
@@ -108,18 +116,22 @@ func getGlobalOptions(ctx *cli.Context) (*profileEntry, error) {
 func profileFromContext(ctx *cli.Context, store bool) (*profileEntry, error) {
 	// Parse the paths of the cert and macaroon. This will validate the
 	// chain and network value as well.
+	tlsCert := []byte("")
 	tlsCertPath, macPath, err := extractPathArgs(ctx)
 	if err != nil {
 		return nil, err
 	}
 
 	// Load the certificate file now. We store it as plain PEM directly.
-	tlsCert, err := ioutil.ReadFile(tlsCertPath)
-	if err != nil {
-		return nil, fmt.Errorf("could not load TLS cert file %s: %v",
-			tlsCertPath, err)
+	// Because we use the system's cert pool, Only attempt to load it
+	// if the file exists.
+	if fileExists(tlsCertPath) {
```

Comment:
This won't work reliably. The `--tlscertpath` flag has a default value. And if the certificate is placed in the default location (even if it's one signed by a CA), this will always be true. I think we need to add a `ctx.GlobalIsSet("tlscertpath")` here as well.